modules/trusted-firmware-m/Kconfig

```diff
@@ -103,6 +103,7 @@ config PM_PARTITION_SIZE_TFM
 	hex
 	prompt  "Memory reserved for TFM" if !TFM_PROFILE_TYPE_MINIMAL
 	default 0x10000 if TFM_PROFILE_TYPE_MINIMAL && TFM_CMAKE_BUILD_TYPE_DEBUG
+	default 0xC000 if TFM_PROFILE_TYPE_MINIMAL && SOC_NRF9160 && BOOTLOADER_MCUBOOT && !SECURE_BOOT
```

Comment:
Curious: Why `!SECURE_BOOT`?

Reply:
The alignment issue is not present there because of the sizes of the bootloader chain.